code/game/objects/items/stacks/wrap.dm

```diff
@@ -94,6 +94,7 @@
 	max_amount = 25
 	resistance_flags = FLAMMABLE
 	grind_results = list(/datum/reagent/cellulose = 5)
+	custom_materials = list(/datum/material/paper = SMALL_MATERIAL_AMOUNT)
```

Comment:
You need to use `mats_per_unit` for stacks, and it should correspond to the amount of materials each additional unit has and not that of the entirety of the stack.